lib/public/views/Runs/Overview/index.js

```diff
@@ -48,7 +48,7 @@ const showRunsTable = (model, runs) => {
             filtersToggleButton(model.runs),
             filtersPanel(model, model.runs, runsActiveColumns),
             h(
-                `.w-60.filters${model.runs.isAnyFilterActive() && !model.runs.areFiltersVisible ? '.display-block' : '.display-none'}`,
+                `.w-60.mh3${model.runs.isAnyFilterActive() && !model.runs.areFiltersVisible ? '.display-block' : '.display-none'}`,
```

Comment:
The idea is good, but maybe you should use baloon instead? You can see the detectors column in the runs overview